src/BlueprintParser.h

```diff
@@ -643,7 +643,7 @@ namespace snowcrash {
                  elementIt != out.node.content.elements().end();
                  ++elementIt) {
 
-                if (elementIt->element == Element::CategoryElement) {
+                if (elementIt->element == Element::CategoryElement && elementSourceMapIt != out.sourceMap.content.elements().collection.end()) {
```

Comment:
Infact, actually having this check will crash the program since not all people use `sourceMap` flag when running the program.

**DO NOT MERGE**